release_notes.txt

```diff
@@ -1,3 +1,75 @@
+################################################################################################
+Release Notes for OpenMDAO 2.3.0
+
+June 12, 2018
+
+New Features:
+--------------
```

Comment:
https://www.pivotaltracker.com/story/show/157854109 User can provide a CSV file to the DOEDriver to run an arbitrary set of cases.

Reply:
https://www.pivotaltracker.com/story/show/158011209 User can set check_partial_options to use CS if alloc=complex, but the component will do FD for check_partials of CS not available.

Reply:
https://www.pivotaltracker.com/story/show/149861856  User can give a case-prefix when the call run_model or run_driver by passing an argument to the function.

Reply:
https://www.pivotaltracker.com/story/show/157854243 should mention option change to 'procs_per_model' from 'parallel'

Reply:
https://www.pivotaltracker.com/story/show/157128289 SimpleGA driver can support models that need more than 1 proc

Reply:
https://www.pivotaltracker.com/story/show/157752792 Beam Example expanded/improved.